Tests/JWTRequestHandlerTests.cs

```diff
@@ -125,7 +125,7 @@ public async Task TestTokenRefreshAsync()
             }
 
             // arrange
-            HttpResponseMessage response401 = new HttpResponseMessage(HttpStatusCode.Unauthorized);
```

Comment:
In another places var->string, here Type->var. For why?

Reply:
It uses 'var' when type is evident